src/Entity/PropertyId.php

```diff
@@ -64,7 +70,7 @@ public function getEntityType() {
 	 * @return string
 	 */
 	public function serialize() {
-		return json_encode( array( 'property', $this->serialization ) );
+		return json_encode( array( 'property', $this->getSerialization() ) );
```

Comment:
Yea, this can be left for later. If we can't change unserialize(), why change serialize().